Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -353,10 +355,33 @@ const ScrollView = React.createClass({
 
   mixins: [ScrollResponder.Mixin],
 
+  _scrollAnimatedValue: (new Animated.Value(0): Animated.Value),
```

Comment:
Is there another way to declare instance variables with createClass that are not nullable? I can't just initialize these values here because otherwise the same object gets used for every instance so I have to do it in componentWillMount. As a workaround I'm just creating these values to be able to make the type not nullable.